banzai_floyds/wavelengths.py

```diff
@@ -266,31 +266,31 @@ def do_stage(self, image):
         orders = np.unique(image.orders.data)
         orders = orders[orders != 0]
         initial_wavelength_solutions = []
-        for order in orders:
-            # copy order centers and get mask for height of a few extract median along axis=0
-            extraction_orders = copy(image.orders)
-            extraction_orders._order_height = self.EXTRACTION_HEIGHT
+        # copy order centers and get mask for height of a few extract median along axis=0
```

Comment:
I know this is just a moved comment, but I feel like it could be rephrased for clarity.